content/numpy/concepts/math-methods/terms/trunc/trunc.md

```diff
@@ -0,0 +1,92 @@
+---
+Title: '.trunc()'
+Description: 'Truncate the decimal part of each element in an array, returning the integer part as a float.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+  - 'NumPy'
+  - 'Formatting'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+**`numpy.trunc()`** truncates the decimal part of each element in a NumPy array, returning the integer part of the number as a float. It is often used when you need to ignore the fractional part without rounding.
+
+## Syntax
+
+```pseudo
+numpy.trunc(x, /, out=None, *, where=True)
+```
+
+- `x`: The input array whose elements you want to truncate.
+- `out`: (Optional) A location where the result is stored. If not provided, a new array is created.
+- `where`: (Optional) A boolean array that specifies which elements to compute for. Default is True for all elements.
+
+It returns a NumPy array containing the truncated values, where each element is the integer part of the input.
```

Comment:
Since the `.trunc()` function can also be used on scalar values, this line can be modified to fit that as well. Example:

"It returns a new array (or a scalar if the input is a single value) with the decimal part truncated from each element, leaving the integer part as a float."